### DISCUSSION TITLE
How to make gauge chart with Vega-Lite

### DISCUSSION THREAD

**Discussion by shenenya** [vega/vega-lite#9597] [Q&A]
How can we make gauge chart with Vega-Lite? Thanks.
<img width="304" alt="截屏2019-12-17下午10 24 15" src="https://user-images.githubusercontent.com/31716017/71008883-519cef00-2124-11ea-9339-a7b794ae2f6a.png">

**Comment by domoritz** [comment]
Please ask questions on stack overflow.

**Comment by jasalt** [comment]
This issue came up when searching solution for same problem.
With vega-lite it might be complicated as it's more about pre-defined visualizations.
Found this Vega tutorial worth reading to accomplish it more low level https://observablehq.com/@mayukobonnefoy/how-to-create-a-gauge-with-vega.

Maybe this issue can be converted to a Github Discussion now that it exists so traffic coming here would be served better.
